cgi/generate_sample_import_file.pl

```diff
@@ -56,16 +56,26 @@
 my $worksheet = $workbook->add_worksheet();
 my %formats = (
 	normal => $workbook->add_format(border => 1, bold => 1),
-	mandatory => $workbook->add_format(border => 1, bold => 1, bg_color => '#aaffcc'),
-	recommended => $workbook->add_format(border => 1, bold => 1, bg_color => '#ccffdd'),
-	optional => $workbook->add_format(border => 1, bold => 1, bg_color => '#eeffee'),
+	mandatory => $workbook->add_format(border => 1, bold => 1, bg_color => '#0f55cc', color => 'white', valign => 'vcenter', align=>'center'),
+	recommended => $workbook->add_format(border => 1, bold => 1, bg_color => '#ccffdd', valign => 'vcenter', align=>'center'),
```

Comment:
Could you also change the recommended and optional cells to a lighter blue and an even lighter blue?

Reply:
Yes, sure.